homeassistant/components/nest/strings.json

```
@@ -9,19 +9,19 @@
       "link": {
         "title": "Link Nest Account",
         "description": "To link your Nest account, [authorize your account]({url}).\n\nAfter authorization, copy-paste the provided pin code below.",
-        "data": { "code": "Pin code" }
+        "data": { "code": "[%key:common::config_flow::data::pin%]" }
       }
     },
     "error": {
       "timeout": "Timeout validating code",
       "invalid_code": "Invalid code",
-      "unknown": "Unknown error validating code",
+      "unknown": "[%key:common::config_flow::error::unknown%]",
       "internal_error": "Internal error validating code"
     },
     "abort": {
-      "already_setup": "You can only configure a single Nest account.",
-      "no_flows": "You need to configure Nest before being able to authenticate with it. [Please read the instructions](https://www.home-assistant.io/components/nest/).",
-      "authorize_url_timeout": "Timeout generating authorize url.",
+      "already_setup": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_flows": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
```

Change the key here as well:
```suggestion
      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
```